selfdrive/car/hyundai/fingerprints.py

```diff
@@ -529,9 +529,6 @@
     (Ecu.fwdRadar, 0x7d0, None): [
       b'\xf1\x00OS__ SCC F-CUP      1.00 1.00 95655-J9200         ',
     ],
-    (Ecu.abs, 0x7d1, None): [
-      b'\xf1\x816V5RAK00018.ELF\xf1\x00\x00\x00\x00\x00\x00\x00',
-    ],
```

Comment:
Neither of our ABS queries return expected responses, unlike other platforms. The other query returns `b'\xf1\x00\x00\x00\x00\x00\x00\x00',` Removing!